firedrake/slate/slac/kernel_builder.py

```diff
@@ -54,7 +54,7 @@ class LocalKernelBuilder(object):
     mesh_layer_sym = ast.Symbol("layer")
 
     # Supported integral types
-    supported_integral_types = [
+    integral_keys = [
```

Comment:
Again, why rename this variable in this changeset?  Especially to something that is less descriptive.